examples/jenkins/master-slave/README.md

```diff
@@ -1,97 +1,37 @@
 Jenkins Master->Slave Example
 =============================
-This sample walks through the process of starting up an OpenShift cluster and deploying a Jenkins Pod in it.
-It also leverage the [kubernetes-plugin](https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin) to manage
-the Jenkins slaves that run as on-demand Pods.
+This sample leverages the [kubernetes-plugin](https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin) to manage
+the Jenkins slaves that run as on-demand Pods.  The kubenetes-plugin are pre-installed into the OpenShift Jenkins Images
+for Centos and RHEL produced by the [OpenShift Jenkins repository](https://github.com/openshift/jenkins).  The OpenShift
+Jenkins repository also produces a [base Jenkins slave image](https://github.com/openshift/jenkins/tree/master/slave-base),
+as well as sample Jenkins slave images for [Maven](https://github.com/openshift/jenkins/tree/master/slave-maven) and
+[NodeJS](https://github.com/openshift/jenkins/tree/master/slave-nodejs) which extend that base Jenkins slave image.
 
-The sample job that this Jenkins example contains will execute an unit test for
-a simple Ruby application. The unit test will be executed in a pod managed by
-the kubernetes plugin.
-
-Because the Jenkins Slave pod needs a Docker image that connects to Jenkins
-when it starts, this sample also provides a template that allows you to convert
-any OpenShift S2I image to a valid Jenkins Slave image.
-
-This template is not required if you already have Docker image that has the
-script that launches the [slave
-agent](https://wiki.jenkins-ci.org/display/JENKINS/Distributed+builds#Distributedbuilds-Launchslaveagentheadlessly)
-as an entrypoint. In that case, you have to create the image stream for this
-image and set the image stream label `role` to `jenkins-slave`.
-
-The [Jenkins image](https://github.com/openshift/jenkins) includes the
-Kubernetes plugin, so it can manage slave pods by default. This tutorial
-includes a template which further customizes the Jenkins image by adding the Git
-plugin and define the sample job used by the tutorial.
+The sample builds upon the [sample job](https://github.com/openshift/origin/blob/master/examples/jenkins/README.md)
+described in this file's parent directory, and leverages the OpenShift Jenkins slave image for NodeJS to launch the sample
+job in a Jenkins slave provisioned as Kubernetes Pod on OpenShift.
```

Comment:
(I guess the base image isn't available as a slave in the default jenkins image configuration.  still it would be worth noting why we're using the nodejs slave)